sdk/src/asset_handlers/bmff_io.rs

```diff
@@ -1066,6 +1066,135 @@ impl AssetIO for BmffIO {
         let vec: Vec<HashObjectPositions> = Vec::new();
         Ok(vec)
     }
+
+    fn remove_cai_store(&self, asset_path: &Path) -> Result<()> {
+        let mut input = File::open(asset_path)?;
+        let size = input.seek(SeekFrom::End(0))?;
+        input.seek(SeekFrom::Start(0))?;
+
+        // create root node
+        let root_box = BoxInfo {
+            path: "".to_string(),
+            offset: 0,
+            size: size as u64,
+            box_type: BoxType::Empty,
+            parent: None,
+            user_type: None,
+            version: None,
+            flags: None,
+        };
+
+        let (mut bmff_tree, root_token) = Arena::with_data(root_box);
+        let mut bmff_map: HashMap<String, Vec<Token>> = HashMap::new();
+
+        // build layout of the BMFF structure
+        build_bmff_tree(
+            &mut input,
+            size as u64,
+            &mut bmff_tree,
+            &root_token,
+            &mut bmff_map,
+        )?;
+
+        // get position of c2pa manifest
+        let (c2pa_start, c2pa_length) =
+            if let Some(c2pa_token) = get_manifest_token(&bmff_tree, &bmff_map) {
+                let uuid_info = &bmff_tree[c2pa_token].data;
+
+                (uuid_info.offset, Some(uuid_info.size))
+            } else {
+                return Ok(()); // no box to remove
+            };
+
+        let mut temp_file = Builder::new()
+            .prefix("c2pa_temp")
+            .rand_bytes(5)
+            .tempfile()?;
+
+        let (start, end) = if let Some(c2pa_length) = c2pa_length {
+            let start = usize::value_from(c2pa_start)
+                .map_err(|_err| Error::BadParam("value out of range".to_string()))?; // get beginning of chunk which starts 4 bytes before label
```

Comment:
This is a pretty obscure error if this ever comes up. It isn't really a bad parameter, more like invalid bmff data?